src/config.rs

```diff
@@ -0,0 +1,35 @@
+use anyhow::Result;
+use serde::{Deserialize, Serialize};
+use std::{fs, path::Path};
+
+use crate::info::utils::info_field::InfoType;
+
+#[derive(Serialize, Deserialize)]
+pub struct ConfigOptions {
+    pub disabled_fields: Option<Vec<InfoType>>,
```

Comment:
I don't believe this needs to be wrapped on an `Option`. If your goal is to avoid breakage when the user doesn't define this value, there should be serde options to fall back to a default.

Also, since `Vec::default()` will always be empty, for now you could probably just derive `Default`.